### DISCUSSION TITLE
v4蒸馏模型的教师模型权重什么时候可以提供了，现在只能下载到学生模型的权重

### DISCUSSION THREAD

**Discussion by lonelyaustralis** [PaddlePaddle/PaddleOCR#12186] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

**Comment by SWHL** [comment, marked as answer]
这里已经提供了：[link](https://github.com/PaddlePaddle/PaddleOCR/blob/230da309e8b4130c9c76536a33c321fc5510c1d8/configs/det/ch_PP-OCRv4/ch_PP-OCRv4_det_cml.yml#L59C19-L59C123)
